4337/src/utils/safe.ts

```diff
@@ -88,6 +88,7 @@ const actionCalldata = (action: MetaTransaction): string => {
 }
 
 export interface RpcProvider extends Provider {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   send(method: string, params: unknown[]): Promise<any>
```

Comment:
This is the interface of a `JsonRpcProvider` in `ethers`, so I just disabled the lint for the `send` function.